### PR TITLE
ui v2: accordion group

### PR DESCRIPTION
**Pull request by danielhochman**
### Description
Add a mutually exclusive expanded accordion group.

### Testing Performed
Storybook.

### CODE REVIEW

File: frontend/packages/core/src/accordion.tsx [hunk 1]
```diff
@@ -77,36 +78,92 @@ export const StyledAccordionSummary = styled(AccordionSummaryBase)({
...
+  ".MuiAccordion-root:before": {
+    display: "none",
+  },
+});
+
+export interface AccordionProps extends Pick<MuiAccordionProps, "defaultExpanded" | "expanded"> {
   title?: string;
   collapsible?: boolean;
-  children: React.ReactNode[];
+  children: React.ReactNode | React.ReactNode[];
```
Comment (dschaller) [inline frontend/packages/core/src/accordion.tsx:98]: `ReactNode` should accept an array of ReactNodes so I think you can do:
```suggestion
  children: React.ReactNode;
```

File: frontend/packages/core/src/accordion.tsx [hunk 2]
```diff
@@ -77,36 +78,92 @@ export const StyledAccordionSummary = styled(AccordionSummaryBase)({
...
+      <StyledAccordionSummary expanded={expanded} collapsible={collapsible} onClick={handleClick}>
         {title}
       </StyledAccordionSummary>
       {children}
     </StyledAccordion>
   );
 };
 
+export interface AccordionGroupProps {
+  children: React.ReactElement | React.ReactElement[];
```
Comment (dschaller) [inline frontend/packages/core/src/accordion.tsx:142]:
```suggestion
  children: React.ReactElement;
```

**Review by dschaller — APPROVED** [review]

File: frontend/packages/core/src/accordion.tsx (hunk 2 above)
Comment (danielhochman) [reply frontend/packages/core/src/accordion.tsx:142]:
```
This JSX tag's 'children' prop expects a single child of type 'ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)>) | (new (props: any) => Component<any, any, any>)>', but multiple children were provided.ts(2746)
```

Comment (danielhochman) [reply frontend/packages/core/src/accordion.tsx:142]: (worked for node but not for element)

Comment (dschaller) [reply frontend/packages/core/src/accordion.tsx]: ah sorry I misread this as `ReactNode`

**Pull request merged**